R/pmax-method.R

```diff
@@ -0,0 +1,22 @@
+#
+# Start Preamble to set up generic pmax
+#
+existingPmaxGeneric <- getGeneric("pmax")
+if (!is.null(existingPmaxGeneric)) {
+  if (!all(existingPmaxGeneric@signature == c("..."))) {
```

Comment:
The `c()` is redundant I believe. I find `any !=` more intuitive/easier to grasp than `!all ==`, but that is clearly personal preference :)
```suggestion
  if (any(existingPmaxGeneric@signature != "...")) {
```